pkg/metrics/metrics.go

```diff
@@ -0,0 +1,40 @@
+// Copyright Contributors to the Open Cluster Management project
+
+package metrics
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promauto"
+)
+
+var (
+	PromRegistry = prometheus.NewRegistry()
+
+	RequestCount = promauto.With(PromRegistry).NewCounterVec(prometheus.CounterOpts{
+		Name: "search_indexer_request_count",
+		Help: "Total requests received by the search indexer (from managed clusters).",
+	}, []string{"managed_cluster_name"})
```

Comment:
Yes, the reason for the separate metric is because this one contains the managed_cluster_name label.  If we add that label to the histogram we will be generating way too much data when managing 1,000 clusters.